packages/next-app/src/app/pool/[poolId]/page.tsx

```diff
@@ -3,7 +3,7 @@ import { MetadataAttributesTable } from "./(components)/MetadataAttributesTable"
 export default function Page({ params }: { params: { poolId: string } }) {
   return (
     <div className="h-full flex-1 py-5 text-white">
-      <MetadataAttributesTable poolId={params.poolId} />
+      <MetadataAttributesTable poolId={params.poolId as `0x${string}`} />
```

Comment:
you could also have changed this in the definition in line `3` instead

Reply:
Thanks, I'll apply the change